test/unit/test-amp-img.js

```diff
@@ -406,7 +407,10 @@ describes.sandboxed('amp-img', {}, (env) => {
     el.getLayoutSize = () => ({width: 300, height: 200});
 
     const impl = new AmpImg(el);
-    const propagateAttributesSpy = sandbox.spy(impl, 'propagateAttributes');
+    const propagateAttributesSpy = sandbox.spy(
```

Comment:
I vaguely recall Justin saying it's problematic that we use this pattern to spy on module functions, but I don't know how problematic or what the workaround would be

Reply:
@jridgewell I remember that too now, what pattern should I use instead here?